src/solvers/sat/satcheck_minisat2.h

```diff
@@ -23,6 +23,12 @@ class Solver; // NOLINT(readability/identifiers)
 class SimpSolver; // NOLINT(readability/identifiers)
 }
 
+struct satcheck_minisat_infot final
```

Comment:
This file will only be included when `SATCHECK_MINISAT2` is defined, hence all other cases will cause linking failures under the subsequent commits.